game-core/src/main/java/games/strategy/triplea/attachments/UnitSupportAttachment.java

```diff
@@ -214,7 +215,19 @@ private void resetNumber() {
     number = 0;
   }
 
-  private void setBonusType(final String type) {
+  private void setBonusType(final String type) throws GameParseException {
+    final String[] s = splitOnColon(type);
+    if (s.length > 2) {
+      throw new GameParseException("bonusType can only have value and count" + thisErrorMsg());
```

Comment:
It might not be clear which value was the offending value, does it make sense to print out the 'pre-split' value?

Reply:
Added.